openslides/motions/serializers.py

```diff
@@ -102,7 +102,7 @@ class WorkflowSerializer(ModelSerializer):
     """
     states = StateSerializer(many=True, read_only=True)
     # The first_state is checked in the update() method
-    first_state = PrimaryKeyRelatedField(queryset=State.objects.all(), required=False)
+    first_state = PrimaryKeyRelatedField(read_only=True)
```

Comment:
We also need some changes in the update method. Maybe our custom update method can be removed and we can also remove this line and set read_only via [Meta option](http://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields).